SPECS-EXTENDED/rubygem-introspection/rubygem-introspection.spec

```diff
@@ -0,0 +1,155 @@
+# Generated from introspection-0.0.2.gem by gem2rpm -*- rpm-spec -*-
```

Comment:
Judging from the `%changelog` this is no longer true and will only cause confusion. Let's either update it to mention the 0.0.4 version or simply remove it completely.

Reply:
fixed